client/cli/src/config.rs

```diff
@@ -188,10 +188,10 @@ pub trait CliConfiguration<DCV: DefaultConfigurationValues = ()>: Sized {
 	///
 	/// Bu default this is retrieved from `KeystoreParams` if it is available. Otherwise it uses
 	/// `KeystoreConfig::InMemory`.
-	fn keystore_config(&self, base_path: &PathBuf) -> Result<KeystoreConfig> {
+	fn keystore_config(&self, base_path: &PathBuf) -> Result<(Vec<String>, KeystoreConfig)> {
 		self.keystore_params()
 			.map(|x| x.keystore_config(base_path))
-			.unwrap_or(Ok(KeystoreConfig::InMemory))
+			.unwrap_or(Ok((Vec::new(), KeystoreConfig::InMemory)))
```

Comment:
```suggestion
			.unwrap_or_else(|| Ok((Vec::new(), KeystoreConfig::InMemory)))
```

Clippy will kill you if it sees it :grin: